grpc/proto/weaviate.proto

```diff
@@ -4,13 +4,14 @@ package weaviategrpc;
 
 import "batch.proto";
 import "search_get.proto";
+import "search_get_v1.proto";
 
 option go_package = "github.com/weaviate/weaviate/grpc/generated;protocol";
 option java_package = "io.weaviate.grpc.protocol";
 option java_outer_classname = "WeaviateProto";
 
-
 service Weaviate {
   rpc Search(SearchRequest) returns (SearchReply) {};
+  rpc SearchV1(SearchRequestV1) returns (SearchReplyV1) {};
```

Comment:
maybe instead of introducing a new V1 method here, we would add?
```
service WeaviateV1 {
  rpc Search(SearchRequestV1) returns (SearchReplyV1) {};
  rpc BatchObjects(BatchObjectsRequest) returns (BatchObjectsReply) {};
}
```

Reply:
but then we couldn't return an error on the old search. If you use `Search` (old python client) on this PR you get an error back that encurages you to update. With your search it would just be a "method not known"